_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -12,92 +12,88 @@ redirect_from:
 
 # Getting Started
 
+- [Prerequisites](#prerequisites)
+- [Build the Application](#build-the-application)
+- [Run the Application](#run-the-application)
+
 This tutorial creates an introductory application that uses Inrupt's
 [JavaScript Client Libraries](https://github.com/inrupt/solid-client-js) to read
-and write your user profile data.
-
-Alternatively, to create a sample application using Inrupt's Solid React SDK,
-refer to Inrupt's [Solid React SDK documentation](https://docs.inrupt.com/developer-tools/javascript/react-sdk/).
+and write to your name to your user profile.
 
 This tutorial uses [npm](https://www.npmjs.com/get-npm) and [Parcel](https://parceljs.org/)
 to run the application locally on `localhost:1234`.
 
 ![Locally Run Getting Started Application]({{site.baseurl}}/assets/img/tutorials/locally-run-application.png)
 
+> You can also try out the application in CodeSandbox.
+>
+> [![Edit on CodeSandbox]({{site.baseUrl}}/assets/img/play-codesandbox.svg)](https://codesandbox.io/s/musing-bouman-n1ghd)
+>
+> Inside CodeSandbox, rather than running the application in the embedded browser, open the application in a new window to be able to log in.
+
 ## Prerequisites
 
-**NOTE:** This tutorial assumes the creation of a testing Pod on
```

Comment:
Moved this to the index.js step since that's where the user would change this value.